lib/MongoDB/ClientSession.pm

```diff
@@ -61,14 +78,24 @@ has options => (
     },
 );
 
+=method server_session
+
+The server session containing the unique id for this session. See
+L<MongoDB::ServerSession> for more information.
+
+=cut
+
 has server_session => (
     is => 'rwp',
     isa => Maybe[InstanceOf['MongoDB::ServerSession']],
```

Comment:
This is a Maybe so that when the session is ended, the server_session can be set to undef and returned to the pool - Something about leaving the reference to the server_session in the ClientSession after it has been returned to the pool makes me think something could go wrong.

Reply:
Ah.  I'd still like to enforce that we start with an actual server_session.  Maybe a BUILD method to ensure it's defined?  Please bounce it off Matt and see what he thinks -- maybe there's some Moo-ish idiom I don't know.

Reply:
I don't think I saw a change to require this via BUILD.  Did I miss something else subtle?